common/src/main/java/com/wynntils/gui/render/buffered/CustomRenderType.java

```diff
@@ -34,9 +34,49 @@ public class CustomRenderType extends RenderType {
                     .setCullState(NO_CULL)
                     .createCompositeState(false));
 
-    private static final Function<ResourceLocation, RenderType> POSITION_COLOR_TEXTURE =
+    public static final RenderType POSITION_COLOR_TRIANGLE_STRIP = RenderType.create(
+            "wynntils_position_color_triangle_strip",
+            DefaultVertexFormat.POSITION_COLOR,
+            Mode.TRIANGLE_STRIP,
+            256,
+            false,
+            false,
+            CompositeState.builder()
+                    .setShaderState(POSITION_COLOR_SHADER)
+                    .setTransparencyState(TRANSLUCENT_TRANSPARENCY)
+                    .setWriteMaskState(RenderStateShard.COLOR_DEPTH_WRITE)
+                    .createCompositeState(false));
+
+    public static final RenderType POSITION_QUAD = RenderType.create(
+            "wynntils_position_texture_quad",
+            DefaultVertexFormat.POSITION_TEX,
+            Mode.QUADS,
+            256,
+            false,
+            false,
+            CompositeState.builder()
+                    .setShaderState(POSITION_COLOR_SHADER)
+                    .setTransparencyState(CustomRenderStateShard.SEMI_TRANSPARENT_TRANSPARENCY)
+                    .setWriteMaskState(COLOR_WRITE)
+                    .createCompositeState(false));
+
+    private static final Function<ResourceLocation, RenderType> POSITION_TEXTURE_QUAD =
             Util.memoize(resource -> RenderType.create(
-                    "wynntils_position_color_texture",
+                    "wynntils_position_texture_quad",
+                    DefaultVertexFormat.POSITION_TEX,
+                    Mode.QUADS,
+                    256,
+                    false,
+                    false,
+                    CompositeState.builder()
+                            .setShaderState(POSITION_TEX_SHADER)
+                            .setTextureState(new TextureStateShard(resource, false, false))
+                            .setTransparencyState(CustomRenderStateShard.SEMI_TRANSPARENT_TRANSPARENCY)
+                            .createCompositeState(false)));
+
+    private static final Function<ResourceLocation, RenderType> POSITION_COLOR_TEXTURE_QUAD =
```

Comment:
why aren't the first two memoized?

Reply:
We don't have to create instances of the render type, if there is no variation in them. But as soon as there is a changing value, like a texture or a different line width, we have to memoize, to cache them (unless we want to fix the values specifically).